subprojects/docs/src/docs/userguide/publishing_ivy.adoc

```diff
@@ -195,3 +195,21 @@ include::{samplesPath}/ivy-publish/java-multi-project/output-ivy.xml[]
 Note that `«PUBLICATION-TIME-STAMP»` in this example Ivy module descriptor will be the timestamp of when the descriptor was generated.
 ====
 
+[[sec:pub_ivy:legacy_migration]]
+== Migrating from legacy Ivy publication
+
+If you are migrating a project that used to rely on the <<artifact_management.adoc#,legacy publishing>> support, you will find below the differences in mapping between the two solutions.
```

Comment:
'will find below the differences in mapping between the two solutions' -> 'will find the differences between the two solutions below'